### ISSUE TITLE
Trying to convert shortnames to image by adding emojione.js along side with emojionearea leads to multiple errors

### ISSUE THREAD

**Issue by IvanShrt** [mervick/emojionearea#142]
Hello! I'm saving emojis as shortnames in my db and trying to convert them to images by this way:
```
$(document).ready(function() {
    // if you save to db with value EmojioneArea saveEmojisAs: 'shortname'
    $(".your-selector-with-shortname-emojis").each(function() {
        $(this).html(emojione.shortnameToImage($(this).html()));
    });
});
```
I works okay but in my console I receive lots of errors (example on picture). I've spent hours trying to fix this by changing code in emojionearea.js but nothing helps. Are there any ideas? I will be very thankful for any help!
![issuepng](https://user-images.githubusercontent.com/16226479/28013389-7f9c804e-6571-11e7-9314-44530ea2021b.png)

I have both:
 ```
<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/emojione/2.2.7/lib/js/emojione.min.js"></script>
<script type="text/javascript" src="js/emojionearea.js"></script>
```
Because the function doesn't work without emojione.js file at all.

P.S. I can't convert them with php, bcz I'm using smarty template engine.

**Comment by mervick** [comment]
1) smarty have api to register functions into smarty: http://www.smarty.net/docsv2/en/api.register.function.tpl
2) emojionearea != emojione, you can read about emojione here https://github.com/emojione/emojione

**Comment by IvanShrt** [comment]
Hi! Thank you very much for your response!
1. Thanks for pointing me, if I will be completely desperate I will try this way. But for now, I'd love to implement it only with JS.
2. Yes, I do realize that :). But what I meant to say is that errors come from emojionearea when emojione .js file is included too...

**Comment by mervick** [comment]
Emojonearea loads emojione by self,
emojionearea change paths of emojione depending on detected version of emojione after it used by `emojione()`,
but in case when user use emojione without emojioneare api, you need to set emojione paths right by yourself.

**Comment by IvanShrt** [comment]
Okay, thank you! That is clear. But how can I do it? What do I need to change in emojionearea to be able to load `src="https://cdnjs.cloudflare.com/ajax/libs/emojione/2.2.7/lib/js/emojione.min.js"` without conflicting it with Emojionearea. I tried to set `useInternalCDN    : false, // Use the self loading mechanism` and deleting the links, but it doesn't help at all. What is the best way to succeed here?

**Comment by mervick** [comment]
Please, create your example here https://jsfiddle.net/ and I will help you

**Comment by IvanShrt** [comment]
https://jsfiddle.net/5suLa2sq/4/ I'm sorry, this is my first fiddle I did. But at least it just shows what do I have in my code. I'm using EmojioneArea v3.1.5 but couldn't find appropriate example, in fiddle it's EmojioneArea v3.1.8

**Comment by IvanShrt** [comment]
It all works quite fine, but as I showed in the picture it got multiple errors in console, and it make the page loading slower a bit

**Comment by mervick** [comment]
In jsfiddle console I don't see any error,
in the case if there were errors, it would have be in the console, 
you can check it on this page (https://jsfiddle.net/5suLa2sq/6/), I added some errors

**Comment by IvanShrt** [comment]
Yes, but there a tons of errors in my browsers when testing, Maybe you know some other way to convert shortnames into images?

**Comment by nasmo** [comment]
Hello, I'm really in trouble with emojioneArea options. I need to use shortnames but when i put it as an option, it doesn't use short names and my form returns an error. Here is my code : 
$(".with-emojis").emojioneArea({
            shortnames: true,
        });
and this is the error : 
Incorrect string value: '\xF0\x9F\xA4\x91' for column 'message'
I'm using laravel and installed emijione and emojioneArea with npm

**Comment by mervick** [comment]
`shortnames` is for parse shortnames,
you need to set `saveEmojisAs` to `shortname`
https://github.com/mervick/emojionearea/blob/7d462eb6911d50bb93fd0e78e06dcaf5fc2a1013/dist/emojionearea.js#L150

**Comment by nasmo** [comment]
It works ! Thanks a lot.

**Comment by nasmo** [comment]
I abuse of your kindness but i have now another problem. I use @emojione($message) to convert shortnames in images but when i use emojioneArea, it applies a zoom on the emojis making them much bigger and blured. Do you know if I have to de-activate anything ?

**Comment by nasmo** [comment]
Just for info, I use christofferok package to display emojis with Laravel

**Comment by OlegShevtsov1** [comment]
Hello.
I installed emijione and emojioneArea with composer in my project framework Laravel

in /resources/views/layouts/app.blade.php I added code:

$(document).ready(function() {
    $("#comment").emojioneArea({
    });
  });

but in bottom,  I have default code from laravel 
<script src="{{ asset('js/app.js') }}"></script>

So, I get an error:  emojioneArea is not a function
But when I comment this line   , 
 emojioneArea is working.

What is the conflict?
Thanks.

p.s.
Maybe my question will be useful.
